### DISCUSSION TITLE
Could not parse nuclei-ignore file: EOF

### DISCUSSION THREAD

**Discussion by Sha0linn** [projectdiscovery/nuclei-templates#7007] [Q&A] [answered]
Quando eu inicio o nuclei ele me retorna esse erro: Could not parse nuclei-ignore file: EOF. Como eu resolvo?

**Comment by princechaddha** [comment, marked as answer]
Would you mind attempting to delete the nuclei-templates directory `sudo rm -r ~/nuclei-templates/`, and then retry running nuclei?

**Comment by olearycrew** [comment]
@Sha0linn please let us know if you can provide the addition info requested. If we don't hear back from you we'll assume this discussion is closed. Thanks!
